docs/querying/sql-functions.md

```diff
@@ -136,6 +136,24 @@ Returns an array of all values of the specified expression.
 
 Concatenates array inputs into a single array.
 
+## ARRAY_OF_DOUBLES_SKETCH
```

Comment:
This needs to go in the `.spelling` file. Search for `ARRAY_CONCAT_AGG`, it would be right after that one.